bindings/matlab/get_dcursor_pos.m

```diff
@@ -0,0 +1,24 @@
+function pos = get_dcursor_pos()
+
+%Get the position of the point under the current displaz cursor focus
+
+fixLdLibPath = '';
+if isunix()
+    % Reset LD_LIBRARY_PATH on unix - matlab adds to it to get the
+    % matlab version of libraries, which tend to conflict with the
+    % libraries used when building displaz.
+    fixLdLibPath = 'env LD_LIBRARY_PATH=""';
+end
+
+[exitStatus, commandStdout] = system(sprintf('%s displaz -querycursor', fixLdLibPath));
+
+if exitStatus ~= 0
+    errorStr = [];
+    errorStr = strcat(errorStr, sprintf('ERROR: Failed to get displaz cursor position.  Stdout was:\n'));
+    errorStr = strcat(errorStr, sprintf('\n%s\n',commandStdout));
+    errorStr = strcat(errorStr, sprintf('\nERROR: Command Line used was:\n'));
+    errorStr = strcat(errorStr, sprintf('\n%s\n',commandLine));
+    error(errorStr);
+end
+
+pos = sscanf(commandStdout,'%f');
```

Comment:
The rest of the displaz interface assumes xyz are rows, so better to transpose this.